respa_admin/templates/respa_admin/resources/_unit_user_list.html

```diff
@@ -0,0 +1,94 @@
+                <div class="col-md-2">
+                    <span>{{ unit_user.first_name }} {{ unit_user.last_name }}</span>
+                </div>
+                <div class="col-md-3">
+                    <span>{{ unit_user.email }}</span>
+                </div>
+                <div class="col-md-4 text-nowrap">
+                    <div class="col-md-4">
+                        <div>
+                            <span class="{% if unit_user.is_staff %}shape-success{% else %}shape-danger{% endif %}"></span>
+                            <span>{% trans 'Staff' %}</span>
+                        </div>
+                        <div>
+                            <span class="{% if authorization.level == authorization.level.admin %}shape-success{% else %}shape-danger{% endif %}"></span>
+                            <span>{% trans 'Admin' %}</span>
+                        </div>
+                    </div>
+                    <div class="col-md-8">
+                        <div>
+                            <span class="{% if authorization.level == authorization.level.manager %}shape-success{% else %}shape-danger{% endif %}"></span>
+                            <span>{% trans 'Manager' %}</span>
+                        </div>
+                        <div>
+                            <span class="{% if user_can_approve %}shape-success{% else %}shape-danger{% endif %}"></span>
+                            <span>{% trans 'Can approve reservations' %}</span>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-md-3">
+                    <div class="icon-text-container">
+                    <svg class="person-edit-icon" width="28px" height="28px" viewBox="0 0 28 28" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+                        <g id="icons/edit-user" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" stroke-linecap="square">
+                            <g id="a-edit-outline-64" stroke="#000000" stroke-width="2">
+                                <path d="M5.25,6.53333333 C5.25,2.92506667 8.188125,0 11.8125,0 C15.436875,0 18.375,2.92506667 18.375,6.53333333 C18.375,10.1416 15.436875,14 11.8125,14 C8.188125,14 5.25,10.1416 5.25,6.53333333 Z" id="Shape"></path>
+                                <path d="M17.5,18.8591375 C15.9526723,18.5699062 14.1147601,18.375 11.9733691,18.375 C6.84324097,18.375 3.45661956,19.4967938 1.63344386,20.3418688 C0.635049078,20.80435 0,21.838075 0,22.9791188 L0,28 L11.9733691,28" id="Shape"></path>
+                                <polygon id="Shape" points="19.3641304 27.5434783 16.625 28 17.0815217 25.2608696 24.8423913 17.5 27.125 19.7826087">
+                                </polygon>
+                            </g>
+                        </g>
+                    </svg>
+                    <h6>
+                        <a href="">{% trans 'Edit permissions' %}</a>
```

Comment:
Hide the `Edit permissions` link if the user is request.user - a user should not be able to edit their own permissions.

This also needs to be enforced in the permission edit functionality in the next issue.